pkg/morph/event/listener.go

```diff
@@ -368,6 +368,11 @@ func (l listener) parseAndHandleNotary(nr *response.NotaryRequestEvent) {
 		return
 	}
 
+	log := l.log.With(
+		zap.String("contract_LE", notaryEvent.ScriptHash().StringLE()),
```

Comment:
I suggest to assume that all string values are `LE`, because it is human readable format by default for Neo related things. Or you really want to emphasize encoding?

Reply:
no, i do not want

just saw "script hash LE"

deleted "LE"